yolov5/val.py

```diff
@@ -156,6 +157,60 @@ def process_batch(detections, labels, iouv):
     return torch.tensor(correct, dtype=torch.bool, device=iouv.device)
 
 
+def get_current_time():
```

Comment:
let's better move this to the tables.py script?

Reply:
Or even to utils? It's not really a table thing no?

Reply:
In the other PR i made a new folder, want to merge this code first to master. 

This code will be moved to time_utils.py